modules/ROOT/pages/coverage-studio-concept.adoc

```diff
@@ -6,51 +6,69 @@ endif::[]
 You can run a coverage report from Anypoint Studio.
```

Comment:
Why would you do that?

Reply:
The overview of the coverage-report purpose is explained at: https://docs.mulesoft.com/munit/latest/munit-coverage-report. The coverage-studio-concept.adoc article is a child of the munit-coverage-report.adoc article.

Reply:
OK! Presumably linked from this one, too.